rmgpy/tools/canteramodel.py

```diff
@@ -208,24 +221,39 @@ def __init__(self, species_list=None, reaction_list=None, canteraFile='', output
         `sensitive_species`: a list of RMG species objects for conductng sensitivity analysis on
         `thermo_SA`: a boolean indicating whether or not to run thermo SA. By default, if sensitive_species is given,
                       only kinetic_SA will be calculated and it must be additionally specified to perform thermo SA.
+        `surface`: a boolean indicating whether or not to run a surface simulation
         """
         self.species_list = species_list
         self.reaction_list = reaction_list
         self.reaction_map = {}
         self.model = ct.Solution(canteraFile) if canteraFile else None
+        self.surface = surface
```

Comment:
Rather than requiring users to pass surface, why don't we just check `self.surface = bool(surface_species_list or surface_reaction_list)`? I think is_surface would also be a more informative name for the attribute.

Reply:
I'd like to keep the name as self.surface because it later holds the actual Cantera Surface object